redhat-developer/app-services-cli
# feat(cluster connect): create service registry connection

`rhoas cluster connect` should create a serviceregistry connection along with KafkaConnection. 

Closes # 

### Verification Steps

1. Run `rhoas kafka use --name <name of kafka instance>`.
2. Run `rhoas service-registry use --id <id of registry instance>`.
3. Running the cluster connect command should create connections from context.
```
./rhoas cluster connect
```
4. Running the cluster connect command with `ignore-context` flag should prompt for values.
5. ```
./rhoas cluster connect --ignore-context
```

### Type of change


### Checklist
